kythe/go/extractors/gcp/bazel/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build: bazel build //kythe/go/extractors/gcp/bazel:artifacts
+# Usage:
+#   This container houses bazel extraction artifacts which are necessary for
+#   extracting a bazel repo.  It includes a script for modifying a repo's
+#   WORKSPACE in /opt/kythe/extractors/setup-bazel-repo.sh, as well as an
+#   installed copy of the kythe repo at /opt/kythe/repo/kythe.
+#
+#   From commandline:
+#     docker run -it --entrypoint "/bin/sh" gcr.io/kythe-public/kythe-bazel-extractor-artifacts
+#
+#   From Google Cloud Build:
+#     - name: 'gcr.io/kythe-public/kythe-bazel-extractor-artifacts'
+#       volumes:
+#         - name: 'kythe_extractors'
+#           path: '/opt/kythe/extractors/'
+#         - name: 'kythe_repo'
+#           path: '/opt/kythe/repo/kythe/'
+
+FROM launcher.gcr.io/google/ubuntu16_04
+
+# TODO(danielmoy): for modifying other bazel repos, use the setup-bazel-repo script.
+ADD kythe/examples/bazel/setup-bazel-repo.sh /opt/kythe/extractors/setup-bazel-repo.sh
+
+# These are instructions for installing bazel from apt-get, but it doesn't work
+# because one of the deps 'openjdk-8-jdk' cannot be found.
+#ENV KYTHE_PRE_DEPS \
```

Comment:
Well actually, now

(1) I did get this working, but

(2) the more I think about it, the less I want to isntall apt-get install bazel anyways, so just gonna go with the wget route to a specific pinned version.

Reply:
yeah, wget and pinning sounds like a safer way to get bazel, especially since kythe may be sensitive to the exact bazel version you pull down.

Reply:
Keep in mind you can also ask apt for a specific version of a package with something like `sudo apt install bazel=0.20.0`.

Reply:
Hmm that's true.

Other random thought - the wget/script install instructions are listed first on bazel.build's installation instructions, if that has any effect.